packages/api/db/migration/20230922175919_add-crop-generated-boolean-to-revenue-type-table.js

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async function (knex) {
+  await knex.schema.alterTable('revenue_type', (table) => {
+    table.boolean('agriculture_associated').defaultTo(true);
+    table.boolean('crop_generated').defaultTo(false);
```

Comment:
Good question about should we use defaults here. I do not know what is right for architecture so I think it will be fine either way. 

But I _thiiiink_  I prefer having required fields since I don't like assuming things. (One persons opinion that makes sense to me : https://stackoverflow.com/a/6409322) 

But also if undefined can sneak through like this mongoose example that could be not great: https://stackoverflow.com/questions/76114812/should-i-set-a-required-field-if-there-is-already-a-default-field#:~:text=If%20we%20try%20to%20save,required%20field%20is%20not%20redundant.